src/main/java/com/arcbees/bourseje/client/admin/dashboard/AdminDashboardView.java

```diff
@@ -52,7 +52,7 @@
     @UiField
     Element currentState;
     @UiField
-    HTMLPanel candidates;
+    SpanElement candidates;
```

Comment:
That's CSS layout stuff that I'll need to play with in order to understand / fix. Add a todo and leave it to me :)